### PR TITLE
[infra] AppVeyor trigger format server when [format] specified in commit message

### PR DESCRIPTION
**Pull request by archibate**
Related issue = #621 

As mentioned in https://github.com/taichi-dev/taichi/issues/621#issuecomment-601998940

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

**Comment by yuanming-hu** [comment]
Thanks! This is a good idea - I'll review it later since I have a few meetings to attend during the day.

**Comment by archibate** [comment]
Btw, does `improve format server UI` also mean to make frontend UI more **good-looking**? I can do some UI works for you with [mdui](https://www.mdui.org/).

**Comment by archibate** [comment]
Taichi gardener triggered! Now it's only to add `|| true`.

**Comment by yuanming-hu** [comment]
> Btw, does `improve format server UI` also mean to make frontend UI more **good-looking**? I can do some UI works for you with [mdui](https://www.mdui.org/).

I think we should go the github action direction (option 1 in #621 ).

**Comment by archibate** [comment]
> This is a great feature! Could you briefly add this to the document so that future developers will know how to benefit from it?

Good idea! Where are `[skip ci]` and CI related stuff documented? We can put it in the same place with it.

**Comment by yuanming-hu** [comment]
> > 
> 
> Good idea! Where are `[skip ci]` and CI related stuff documented? We can put it in the same place with it.

That would be awesome!

File: docs/contributor_guide.rst [hunk 1]
```diff
@@ -73,6 +73,27 @@ Reviewing & PR merging
...
+- CI, or Continuous Integration, will build and test your commits in PR against different environments to see if there is an error.
+- Currently Taichi uses `"Travis CI" <https://travis-ci.org>`_ and `"AppVeyor" <https://www.appveyor.com>`_.
+- CI will be triggered everytime you push a commit or commits to an opening PR.
+- You can add ``[skip ci]`` to your commit message to avoid triggering CI. e.g. ``[skip ci] your commit message``
+- The checker on the right of commit hash means CI passed, and cross means CI failed.
+
+How to use the format server
+----------------------------
+
+- You can run ``ti format`` in the command line to format the codes, i.e. enforce coding style.
```
Comment (archibate) [inline docs/contributor_guide.rst:88]: Where is `coding style` doced? We can have a ref here.

**Pull request merged**